web/app/themes/radcliffe/header.php

```diff
@@ -2,46 +2,49 @@
 
 <html <?php language_attributes(); ?>>
 
-<head profile="http://gmpg.org/xfn/11">
+<head>
 
     <meta http-equiv="Content-Type" content="<?php bloginfo('html_type'); ?>; charset=<?php bloginfo('charset'); ?>"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=yes">
 
     <?php wp_head(); ?>
 
 </head>
 
 <body <?php body_class(); ?>>
 
+<nav aria-label="Skip Links" class="off-screen"><a href="#maincontent" class="maincontent skip-link">Skip to main content</a></nav>
+
 <div class="header-search-block section hidden">
 
     <div class="section-inner">
 
         <form class="search-form" method="get" action="<?php echo esc_url(home_url('/')); ?>">
-            <label class='search-term-input-label' for="s">Search for: </label>
-            <input type="search" placeholder="enter search term" name="s" id="s" class="search-term-input"/>
+            <label class='search-term-input-label' for="s">Search this website for: </label>
+            <input type="search" name="s" id="s" class="search-term-input no-focus"/>
```

Comment:
Why was the placeholder text removed?

Reply:
From https://knowbility.org/blog/2019/animating-form-labels/ :

"Placeholder text in forms might be thought of as a user experience add-on, but its real-world use is problematic. While there are several issues with placeholder text, a few points to note are:
•	The text might not be translated by auto-conversion tools such as Google Translate.
•	The default color of placeholder text might be too low in contrast for legibility.
•	The text disappears when users start typing, and then people can forget the input’s purpose, especially if the label is visually hidden."

The search instances have been problematic – too complex – for me. Hoping you or Rob can sort them out, perhaps using the suggested “animated form label”